sdk/servicebus/service-bus/test/internal/linkentity.unittest.spec.ts

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
```

Comment:
Filename - `test/internal/linkentity.unittest.spec.ts`

Are you planning to standardize on `.unittest.`?

Reply:
I was considering it - how do you feel about that? @ramya-rao-a had mentioned that there was a possibility of adding more files into internal that were not unit tests so I was trying to get ahead of that, at least with a new file (and probably the older ones).

Reply:
No strong feelings. We can standardize this across the repo if we are doing it.